core/src/main/java/org/apache/iceberg/FileScanTaskParser.java

```diff
@@ -33,6 +33,8 @@ public class FileScanTaskParser {
   private static final String SCHEMA = "schema";
   private static final String SPEC = "spec";
   private static final String DATA_FILE = "data-file";
+  private static final String START = "start";
```

Comment:
nit: do we want to call this `offset` to align with how the field is named in `SplitScanTask`?

Reply:
I call it `start` to match the `FileScanTask/ContentScantask` method name.